### ISSUE TITLE
RBD: Volume name requirements 

### ISSUE THREAD

**Issue by tuxity** [codedellemc/libstorage#417]
# Summary
Volume name regex is to restrictive, and doesn't include the dash `-` which is a common character. There is no error message which indicate us which RR is returning this error 500

# Bug Reports
This section is for issues that relate to discovered problems or bugs.

## Version
Please paste the output of `rexray version`. For example:

```shell
$ rexray version
REX-Ray
-------
Binary: /opt/mesosphere/packages/rexray--c53fc96f1760891c187278a718ef5f96a8d79440/bin/rexray
Flavor: client+agent+controller
SemVer: 0.7.0+6+dirty
OsArch: Linux-x86_64
Branch: release/0.8.0
Commit: a5420b56168f783f2451eda98be33d60c55d3144
Formed: Sat, 11 Feb 2017 01:18:19 CET

libStorage
----------
SemVer: 0.4.0+245
OsArch: Linux-x86_64
Branch: (detached from 1040f4c
Commit: 1040f4c41256950ed7cd82af2d001b0f73cbc5df
Formed: Sat, 11 Feb 2017 01:17:19 CET
```

## Expected Behavior
I should be able to create a volume name with a dash `my-test-volume`

## Actual Behavior
Actually RR return my an error 500 in the logs when I put dashes in my volume name

**Comment by codenrhoden** [comment]
The RBD driver is currently enforcing the name of a volume to be `^\w+$`, which is is also `[a-zA-Z0-9_]. This isn't documented anywhere, and excludes the use of a dash (`-`), which is quite common.

I need to revisit this, relax the requirements where I can, and document what is allowed.  Thanks for the heads up, @tuxity!

**Comment by akutz** [comment]
Hi @tuxity,

I need to add an issue template for libStorage. For now can you refer to https://github.com/codedellemc/rexray/issues/new and how to file issues to see if you need to update this?

**Comment by tuxity** [comment]
@akutz sure, gonna edit it

**Comment by akutz** [comment]
@tuxity,

No need at this point. @codenrhoden responded and is aware of the actual issue. I wasn't so your description wasn't sufficient for me.

**Comment by akutz** [comment]
Hi @tuxity,

The reason I asked was because I was wondering what the error actually was, what the logs said, what your config was, and what command you used.

**Comment by codenrhoden** [comment]
Sorry @akutz, I should have given you a heads up. This issue was created at my request, based on conversation in #project-rexray on Slack.

**Comment by akutz** [comment]
Hi @codenrhoden,

I saw the discussion, but I wasn't paying close attention. My only point was that the issue itself didn't even reference the driver, so it was *really* vague :)

**Comment by cduchesne** [comment]
merged #425
